client/reader/recommendations/index.js

```diff
@@ -38,5 +38,33 @@ export default function() {
 			readerController.removePost,
 			readerController.sidebar,
 			controller.recommendedPosts );
+		page( '/recommendations/cold1w',
```

Comment:
hrm. maybe we could bundle this set up and use page.apply to call it for each path variant? like 
```
forEach( [ path, path, path ], 
    ( path ) => { 
        page.apply( page, [ 
            path, 
            readerController.preloadReaderBundle,
            readerController.loadSubscriptions,
            readerController.updateLastRoute,
            readerController.removePost,
            readerController.sidebar,
            controller.recommendedPosts
] ) } )
```